boards/DCM/Src/App/App_SetPeriodicCanSignals.c

```diff
@@ -0,0 +1,110 @@
+#include "App_SharedSetPeriodicCanSignals.h"
+#include "App_SetPeriodicCanSignals.h"
+#include "App_InRangeCheck.h"
+#include "configs/App_TorqueRequestThresholds.h"
+
+STATIC_DEFINE_APP_SET_PERIODIC_CAN_SIGNALS_IN_RANGE_CHECK(DcmCanTxInterface)
+
+// TODO: Implement PID controller to maintain DC bus power at 80kW
+// #680
+void App_SetPeriodicCanSignals_TorqueRequests(const struct DcmWorld *world)
+{
+    struct DcmCanRxInterface *can_rx = App_DcmWorld_GetCanRx(world);
+    struct DcmCanTxInterface *can_tx = App_DcmWorld_GetCanTx(world);
+
+    // Regen allowed when braking or (speed > 5kmh and AIRs closed)
+    const float regen_threshold_kph = 5.0f;
+    const bool  is_every_air_closed =
+        (App_CanRx_BMS_AIR_STATES_GetSignal_AIR_POSITIVE(can_rx) ==
+         CANMSGS_BMS_AIR_STATES_AIR_POSITIVE_CLOSED_CHOICE) &&
+        (App_CanRx_BMS_AIR_STATES_GetSignal_AIR_NEGATIVE(can_rx) ==
+         CANMSGS_BMS_AIR_STATES_AIR_NEGATIVE_CLOSED_CHOICE);
+    const bool is_vehicle_over_regen_threshold =
+        (App_CanRx_FSM_WHEEL_SPEED_SENSOR_GetSignal_LEFT_WHEEL_SPEED(can_rx) >
+         regen_threshold_kph) &&
+        (App_CanRx_FSM_WHEEL_SPEED_SENSOR_GetSignal_RIGHT_WHEEL_SPEED(can_rx) >
+         regen_threshold_kph);
+    const bool is_regen_allowed =
+        is_vehicle_over_regen_threshold && is_every_air_closed;
+
+    const float regen_paddle_percentage =
+        (float)App_CanRx_DIM_REGEN_PADDLE_GetSignal_MAPPED_PADDLE_POSITION(
+            can_rx);
+    float torque_request;
+
+    /* Calculating the torque request
+     *
+     * 1) If regen. braking is on, use the regen. paddle percentage
```

Comment:
Regen will suffice for the following comments here
```suggestion
     * 1) If regen is on, use the regen paddle percentage
```